lib/membrane_rtc_engine/endpoints/hls_endpoint.ex

```diff
@@ -115,6 +115,14 @@ if Enum.all?(
                   may help players achieve better UX.
                   """
                 ],
+                partial_segment_duration: [
+                  spec: SegmentDuration.t() | nil,
+                  default: SegmentDuration.new(Time.milliseconds(500), Time.milliseconds(550)),
```

Comment:
I am not sure if this should be turned on by default.

Reply:
I will set it to nil to remember to fix version without compositor later on.

Reply:
I'll change that.